src/components/clusters/ClustersTable.tsx

```diff
@@ -64,6 +66,18 @@ const ClustersTable: React.FC<ClustersTableProps> = ({ rows, deleteCluster }) =>
     status: [],
   });
 
+  React.useEffect(() => {
+    const marshalled = window.sessionStorage.getItem(STORAGE_KEY_CLUSTERS_FILTER);
+    if (marshalled) {
+      const parsed = JSON.parse(marshalled);
+      setFilters(parsed);
```

Comment:
Wrap this in try/catch to handle JSON.parse errors since the string in the storage can be manipulated so we're not sure we always get the string we expect.

Reply:
Plenty of other things can be hacked and cause subsequently application failure. But no problem with adding it.

Reply:
Without explicit handling, we would learn about it via sentry